src/components/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Footer.css';
+
+const footerContent = {
+  name: 'Coffeeshot',
+  email: 'coffeeshot@coffeeshot.cz',
+  phone: '732 111 222 333'
+}
+
+class Footer extends React.Component {
```

Comment:
jen pro info, nemusis to predelavat - urcite by tohle mohla byt i pure function :-) he to jednoducha componenta.